src/tagmanager/tm_ctags.c

```diff
@@ -241,6 +241,29 @@ void tm_ctags_clear_ignore_symbols(void)
 }
 
 
+static void replace_str(gchar **where, const gchar *what, guint what_len,
+	const gchar *replacement, guint replacement_len)
```

Comment:
Just a remark, don't change anything for me here: this "API" is quite terrible (not all string args have a corresponding length, and they are still not allowed to lack a terminator anyway), but I guess it makes sense for how it's used (saves some `strlen()`s if the anon name is used more than once).

Reply:
Don't worry, I'm not going to use it as a base for some general-purpose string utility library ;-)